aiida/tools/dbimporters/baseclasses.py

```diff
@@ -209,6 +209,7 @@ def __init__(self, db_name=None, db_uri=None, id=None,
         :param extras: a dictionary with some extra parameters
             (e.g. database ID number)
         :param uri: URI of the structure (should be permanent)
+        :param lic: the license
```

Comment:
This seems to be unused as of now. I suggest taking the ``license`` from keyworded arguments. If it is not defined there, the ``self._license`` is taken from ``_license``.

Reply:
This is in fact what is done now. Stale comment. Fixed.